src/software/RigCalibration/CMakeLists.txt

```diff
@@ -17,9 +17,16 @@ if(OpenMVG_BUILD_VOCTREE)
 endif(OpenMVG_BUILD_VOCTREE)
 
 # Camera calibration
-if(OpenCV_FOUND)
-  message(STATUS "OpenCV found, will build openMVG_main_cameraCalibration")
+if(OpenCV_FOUND AND Boost_FOUND)
+  message(STATUS "OpenCV & Boost found: build openMVG_main_cameraCalibration")
   add_executable(openMVG_main_cameraCalibration main_cameraCalibration.cpp)
-  target_link_libraries(openMVG_main_cameraCalibration ${OpenCV_LIBRARIES})
+  target_link_libraries(openMVG_main_cameraCalibration
+    PUBLIC
+    openMVG_dataio
+    openMVG_image
+    ${OpenCV_LIBRARIES}
+    ${Boost_LIBRARIES}
+    )
+  target_include_directories(openMVG_main_cameraCalibration PUBLIC ${Boost_INCLUDE_DIRS} ${OpenCV_INCLUDE_DIRS})
```

Comment:
currently we have
```
CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
```
so if we decide to move to a more recent version we have to modify that too. And at that point we should upgrade everything to a proper usage of `target_link_libraries` (maybe as a different issue/PR)